app/code/Magento/MediaStorage/Model/File/Storage/MediaStorageConfig.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MediaStorage\Model\File\Storage;
+
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Serialize\SerializerInterface;
+use Magento\MediaStorage\Model\File\Storage;
+
+/***
+ * Class MediaStorageConfig
+ *
+ * @package Magento\MediaStorage\Model\File\Storage
+ */
+class MediaStorageConfig
+{
+    const CACHE_ID = 'config';
+
+    /**
+     * Config cache file path
+     *
+     * @var string
+     */
+    protected $cacheFilePath;
```

Comment:
All new properties have to be private or public. In this case it's better to change visibility to private